packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoWebsocketStatusBadge.tsx

```diff
@@ -25,16 +37,23 @@ export const WfoWebsocketStatusBadge = () => {
             }
         >
             <WfoHeaderBadge
-                color={theme.colors.ghost}
                 textColor={theme.colors.shadow}
                 iconType={() =>
                     websocketConnected ? (
-                        <WfoStatusDotIcon color={theme.colors.success} />
+                        <WfoBoltFill color={theme.colors.success} />
                     ) : (
-                        <WfoXCircleFill color={theme.colors.danger} />
+                        <WfoBoltSlashFill color="yellow" />
```

Comment:
Must: Try to stick with the colors in the theme. The color `warning` or `warningText` from the theme are quite yellow to me too.

Reply:
Fair enough. This is still under debate with @wouter1975 though so it's going to change